tic_tac_toe/TicTacToe.css

```diff
@@ -0,0 +1,97 @@
+.Firstline {
+    display: flex;
+}
+.twoline {
+    display: flex;
+}
+.Threeline {
+    display: flex;
+}
+
+.Sell {
+    position: relative;
+    width: 100px;
+    height: 100px;
+    border: 1px solid grey;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+}
+
+input.o_1_selector{
+    display: none;
+}
+input.x_1_selector{
+    display: none;
+}
+label.x_o_label{
+    cursor: pointer;
+    font-size: 50px;
+    margin: 10px;
+    opacity: 0.3;
+}
+.click_x{
+    display: none;
+    position: absolute;
+    top: 0;
+    left: 0;
+    bottom: 0;
+    right: 0;
+    background-color: rgb(255, 5, 5);
+    font-size: 100px;
+    font-weight: bold;
+}
+input.x_1_selector:checked + .click_x{
```

Comment:
#x_1:checked ~ .Sell > [for="x_1"] ~ .click_x,
#x_2:checked ~ .Sell > [for="x_2"] ~ .click_x,
#x_3:checked ~ .Sell > [for="x_3"] ~ .click_x {
    display: block;
    display: flex;
    align-items: center;
    justify-content: center;
}